pyface/ui/qt4/tasks/main_window_layout.py

```diff
@@ -154,9 +154,6 @@ def set_layout(self, layout):
                     sublayout, q_dock_area, _toplevel_call=False
                 )
 
-        # Remove the fixed sizes once Qt activates the layout.
-        QtCore.QTimer.singleShot(0, self._reset_fixed_sizes)
-
```

Comment:
can you undo the removals and add if/else clauses to not change the behavior on qt4? like in #535 . (i haven't checked the test suite but i'd wager reverting the changes would fix the test suite)

Not sure how to write a regression test for this though.

Reply:
i restored the removed code and moved into a qt4 conditional branch. but i'm still not sure how to add a regression test for this.